ga4gh/datamodel/datasets.py

```diff
@@ -190,3 +237,70 @@ def _setMetadata(self):
                 except KeyError as err:
                     raise exceptions.MissingDatasetMetadataException(
                         metadataFileName, str(err))
+
+
+class AbstractOntology:
+    """
+    The base class of storing an ontology
+    """
+
+    def __init__(self):
+        self.nameById = dict()
+        self.idByName = dict()
+
+    def add(self, id_, name):
+        self.nameById[id_] = name
+        self.idByName[name] = id_
+
+    def getId(self, name):
+        return self.idByName[name]
+
+    def getName(self, id_):
+        return self.nameById[id_]
+
+
+class AbstractOntologies(object):
```

Comment:
How does this relate to the AbstractOntology class above? One of these should probably be deleted.